### ISSUE TITLE
Skip identical frames while muxing seamless branching Blurays

### ISSUE THREAD

**Issue by hubblec4** [mbunkus/mkvtoolnix#2051]
Hi Mosu
For a while we spoke about this topic we email. This is for the remembering on it.

**Comment by mbunkus** [comment]
Thanks. Do you have any information about the following:

1. How many frames may overlap? I guess this is explicitly mentioned in the specs.
2. Are only video tracks affected? Or can this happen for audio and subtitle tracks, too?
3. Can you name a Blu-ray where this is used? I'd like to buy one for testing purposes.

**Comment by hubblec4** [comment]
1. I don't know, but I guess it is only 1 frame every time. But for the Audios the overlap time is very different.

2. I'm not sure but it seems to me that only the Audios affected. Subtitles no, I have never seen in the eac3to log a process for skipping frames for a subtitle.

3. 
- Terminator 2: 3 Editions, skipping frames for AC3 and DTS 
- Bastille Day: 1 Edition but with multiple m2ts. Important 398 fake mpls; 00371.mpls is the correct one
- Rattatouille: Multi-Angle
- Salt: 3 Editions
- Alien Quadrologie: each 2 Editions
- Illuminati: 2 Editions
- Total Recal 2012: 2 Editions

I add two eac3to-demux-log files.
[Bastille Day.zip](https://github.com/mbunkus/mkvtoolnix/files/1165726/Bastille.Day.zip)
[Terminator2.zip](https://github.com/mbunkus/mkvtoolnix/files/1165729/Terminator2.zip)

**Comment by mbunkus** [comment]
Thanks, that's very helpful. I think I'll buy Terminator 2; that one seems to have it all.

**Comment by mbunkus** [comment]
Hmm, so many different editions of T2. Do you mean:

1. [Special Edition/Digitally Remastered](https://www.amazon.de/Terminator-Special-Digital-Remastered-Blu-ray/dp/B073ZJCSMZ/ref=sr_1_3?s=dvd&ie=UTF8&qid=1500649233&sr=1-3&keywords=terminator+2)
2. [Special Edition](https://www.amazon.de/Terminator-Blu-ray-Special-Arnold-Schwarzenegger/dp/B0036GLFPY/ref=sr_1_4?s=dvd&ie=UTF8&qid=1500649233&sr=1-4&keywords=terminator+2)

…and I already ordered 2. back in February, Amazon just told me. I totally forgot 😃

**Comment by hubblec4** [comment]
>I think I'll buy Terminator 2; that one seems to have it all.

Yes it is my favorite seamless branching test Bluray. Many streams avalible for testing.

> Hmm, so many different editions of T2. Do you mean:

I had bought myself the Steel-Edition for a long time. Im not sure what the content of the other Editions are.
The special in the Steel-Edition is the 3. Edition of T2. It is the T-1000 edition which has an alternative end, and in the Bluray Menu you need a "special code" for activating these edition.
I have an iso from this BD WITH encryption(1:1 copy with ImgBurn), if you like I will share it.

**Comment by hubblec4** [comment]
this one
https://www.amazon.de/Terminator-Steelbook-Blu-ray-Arnold-Schwarzenegger/dp/B002HLJ0DI/ref=sr_1_5?s=dvd&ie=UTF8&qid=1500656735&sr=1-5&keywords=terminator+2+steelbook

**Comment by remuxer32** [comment]
:+1:

**Comment by mbunkus** [comment]
This project has been moved to [GitLab](https://gitlab.com/mbunkus/mkvtoolnix/). The corresponding issue is [here](https://gitlab.com/mbunkus/mkvtoolnix/issues/2051).
